ui/src/components/select/QasSelect.vue

```diff
@@ -303,6 +355,40 @@ export default {
       if (popupContentElement) {
         popupContentElement.classList.toggle('qas-select__is-fetching', force)
       }
+    },
+
+    setDefaultOption () {
+      const modelValue = this.attributes.emitValue
+        ? this.options[0].value
+        : this.options[0]
+
+      this.$emit('update:modelValue', modelValue)
+    },
+
+    getFilteredBadgeList ({ label, value, disable, caption, ...rest }) {
+      const badgeList = Object.entries(rest).map(([key, value]) => ({ [key]: value }))
+
+      return badgeList.filter(item => Object.keys(this.badgeProps).includes(Object.keys(item)[0]))
+    },
+
+    getBadgeProps (badge) {
+      const model = Object.keys(badge)[0]
+
+      return this.badgeProps[model](badge[model])
```

Comment:
pq essa posição 0?

Reply:
como faço um v-for, eu passo a badge em questao pro método, ex: `{ isTester: true }`, ai preciso pegar a key (`isTester`) para as validaçòes